frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -161,6 +152,109 @@ export const FormComponentConfig = ({
+            </StudioCard>
+          ) : (
+            <StudioProperty.Button
+              className={classes.gridButton}
+              icon={<PlusCircleIcon />}
+              onClick={() => setShowGrid(true)}
+              property={t('ux_editor.component_properties.grid')}
+            />
+          )}
+        </>
+      )}
+      {/** String properties */}
+      {stringPropertyKeys.map((propertyKey) => {
+        const isSortOrder = propertyKey === 'sortOrder';
+        const isLayout = propertyKey === 'layout';
+
+        return (
+          <CollapsiblePropertyEditor
+            key={propertyKey}
+            label={t(`ux_editor.component_properties.${propertyKey}` as TranslationKey)}
+          >
+            <EditStringValue
+              component={component}
+              handleComponentChange={handleComponentUpdate}
+              propertyKey={propertyKey}
+              helpText={isLayout || isSortOrder ? '' : properties[propertyKey]?.description}
+              enumValues={properties[propertyKey]?.enum || properties[propertyKey]?.examples}
+            />
+          </CollapsiblePropertyEditor>
+        );
+      })}
+      {/** Array properties with enum values) */}
+      {arrayPropertyKeys.map((propertyKey) => {
+        const isShowValidations = propertyKey === 'showValidations';
+        const commonProps = {
+          component,
+          handleComponentChange: handleComponentUpdate,
+          propertyKey,
+          key: propertyKey,
+          helpText: isShowValidations ? '' : properties[propertyKey]?.description,
+          enumValues: properties[propertyKey]?.items?.enum,
+          multiple: true,
+        };
+        return isShowValidations ? (
+          <CollapsiblePropertyEditor
+            key={propertyKey}
+            label={t('ux_editor.component_properties.showValidations')}
+          >
+            <EditStringValue {...commonProps} />
+          </CollapsiblePropertyEditor>
+        ) : (
+          <EditStringValue {...commonProps} />
+        );
+      })}
+      {/** Number properties (number and integer types) */}
+      {numberPropertyKeys.map((propertyKey) => {
+        const isPreselectedOptionIndex = propertyKey === 'preselectedOptionIndex';
+        const commonProps = {
+          component,
+          handleComponentChange: handleComponentUpdate,
+          propertyKey,
+          key: propertyKey,
+          enumValues: properties[propertyKey]?.enum,
+          helpText: isPreselectedOptionIndex
+            ? t('ux_editor.component_properties.preselected_help_text')
+            : undefined,
+        };
+        return isPreselectedOptionIndex ? (
+          <CollapsiblePropertyEditor
+            key={propertyKey}
+            label={t('ux_editor.component_properties.preselectedOptionIndex_button')}
+          >
+            <EditNumberValue {...commonProps} />
+          </CollapsiblePropertyEditor>
+        ) : (
+          <EditNumberValue {...commonProps} />
+        );
```

Comment:
I believe the `CollapsiblePropertyEditor` component should wrap all numbers

```suggestion
        return (
          <CollapsiblePropertyEditor
            key={propertyKey}
            label={t('ux_editor.component_properties.preselectedOptionIndex_button')}
          >
            <EditNumberValue {...commonProps} />
          </CollapsiblePropertyEditor>
        );
```

Reply:
I have not fixed this yet, I am not sure , maybe if we will wrap all numbers with `CollapsiblePropertyEditor` we need to make the label :
 `label={t('ux_editor.component_properties.preselectedOptionIndex_button')}` to 
 dynamic , something  like this maybe? :
 ` label={t(`ux_editor.component_properties.${propertyKey}` as TranslationKey)} `